tests/mockConverter.test.ts

```diff
@@ -546,7 +546,7 @@ export const aPatchBriefDtoAPI = (overrides?: Partial<PatchBriefDto>): PatchBrie
         const expectedString = `
 export const aCreateServiceDtoAPI = (overrides?: Partial<CreateServiceDto>): CreateServiceDto => {
   return {
-    serviceType: aServiceTypeBasicDtoAPI(),
+    serviceType: overrides?.hasOwnProperty('serviceType') ? overrides?.serviceType : aServiceTypeBasicDtoAPI(),
```

Comment:
it would be interesting to test the execution of this typescript actually. This way you would have been able to reproduce the maximum call stack error.
But this can be done later, you've already spent too much time on this project